scripts/install.sh

```diff
@@ -259,6 +259,19 @@ function init() {
     fi
 
     wait_for_nodes
+
+    # once the node is up, apply the master label only on primaries
+    # get the kubernetes version
+    local kubernetes_version=$(kubectl version --short | grep -i server | awk '{ print $3 }' | sed 's/^v*//')
+    semverParse $kubernetes_version
+    KUBERNETES_CURRENT_VERSION_MAJOR="$major"
+    KUBERNETES_CURRENT_VERSION_MINOR="$minor"
+
+    if [ $KUBERNETES_CURRENT_VERSION_MAJOR -ge 1 ] && [ $KUBERNETES_CURRENT_VERSION_MINOR -ge 24 ]; then
```

Comment:
do you need to check this or can you just always apply this label?

Reply:
well, i would like to check this instead of applying blindly. lmk wdyt

Reply:
if you want to leave this code in i would prefer that you not repurpose the global variables. please either define local vars or use $major and $minor directly.

Reply:
will use $major $minor directly..